src/renderer/wg_engine/tvgWgRenderer.cpp

```diff
@@ -314,38 +305,15 @@ bool WgRenderer::target(uint32_t* buffer, uint32_t stride, uint32_t w, uint32_t
 
 
 // target for native window handle
-bool WgRenderer::target(void* window, uint32_t w, uint32_t h)
+bool WgRenderer::target(void *disp_inst, void *wind_serf, uint32_t w, uint32_t h)
```

Comment:
I don't know much about usage/convention of void pointer.
Maybe this type is necessary for wg code, otherwise i'd use explicit type.

The code convention also changed, it should be `void* disp_inst, void* wind_serf`?

Reply:
> Maybe this type is necessary for wg code, otherwise i'd use explicit type

Different patforms used different types for native window description. But in most cases there are always 2 pointers, like pointer to display and window
void* is the most universal type to support all platforms